src/main/java/org/jvnet/hudson/plugins/thinbackup/ThinBackupPluginImpl.java

```diff
@@ -16,26 +16,23 @@
  */
 package org.jvnet.hudson.plugins.thinbackup;
 
-import antlr.ANTLRException;
-import hudson.Plugin;
-import hudson.scheduler.CronTab;
-import hudson.util.FormValidation;
+import hudson.Extension;
+import hudson.ExtensionList;
 import java.io.File;
 import java.util.logging.Level;
 import java.util.logging.Logger;
-import java.util.regex.Pattern;
-import java.util.regex.PatternSyntaxException;
+import jenkins.model.GlobalConfiguration;
 import jenkins.model.Jenkins;
+import net.sf.json.JSONObject;
 import org.apache.commons.lang.StringUtils;
+import org.jenkinsci.Symbol;
 import org.jvnet.hudson.plugins.thinbackup.utils.EnvironmentVariableNotDefinedException;
 import org.jvnet.hudson.plugins.thinbackup.utils.Utils;
-import org.kohsuke.stapler.QueryParameter;
 import org.kohsuke.stapler.StaplerRequest;
-import org.kohsuke.stapler.StaplerResponse;
 
-public class ThinBackupPluginImpl extends Plugin {
-
-    private static final int VERY_HIGH_TIMEOUT = 12 * 60;
+@Extension
+@Symbol("ThinBackup")
+public class ThinBackupPluginImpl extends GlobalConfiguration {
```

Comment:
See https://github.com/jenkinsci/archetypes/blob/e92ac3efd553e4e96a2ff8eb51b15cf9301191a2/global-configuration/src/main/resources/archetype-resources/src/test/java/SampleConfigurationTest.java for how to write a functional test for a `GlobalConfiguration`.

Reply:
That was a very nice hint to get started with testing the UI! Thanks a lot.